core/src/main/kotlin/ch/ergon/dope/resolvable/fromable/UseIndex.kt

```diff
@@ -20,7 +20,11 @@ class IndexReference(
     private val indexName: String? = null,
     private val indexType: IndexType? = null,
 ) : Resolvable {
-    override fun toDopeQuery(manager: DopeQueryManager) = DopeQuery(formatIndexToQueryString(indexName, indexType?.type), emptyMap())
+    override fun toDopeQuery(manager: DopeQueryManager) = DopeQuery(
+        queryString = formatIndexToQueryString(indexName, indexType?.type),
+        parameters = emptyMap(),
```

Comment:
can we maybe make these _emptyMaps_ and _emptyList_ as default values?

Reply:
done, `DopeQuery` can now be initialized with only the `queryString` and the parameters will be empty.